tests/blackbox/package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "tests-blackbox",
 	"private": true,
+	"version": "0.0.0",
```

Comment:
Out of curiosity why the need to add a dummy version here? 🤔

Reply:
Good question! Because of a [bug](https://github.com/changesets/changesets/issues/835) in `changesets` which would prevent us from using the [prerelease mode](https://github.com/changesets/changesets/blob/main/docs/prereleases.md).